### DISCUSSION TITLE
GPU measurements?

### DISCUSSION THREAD

**Discussion by XeLoRy** [opcm/pcm#594] [Q&A] [answered]
Hello!
I used to work with Intel Power Gadget a while ago, and it was very useful to get the GPU-integrated metrics when available on the computer. I didn't see anything GPU related measure in PCM (especially to get energy metrics). Did I miss something? If not, what's the strategy with Intel to get Intel GPU data?
THanks!

**Comment by rdementi** [comment, marked as answer]
I think we can add PP0 and PP1 domain power metrics to pcm. PP1 domain includes integrated GPU power.

**Reply by XeLoRy** [reply]
Thanks for such reactivity ;) I will look at this!

**Reply by ghost** [reply]
do you plan to integrate it into a future release?

**Reply by rdementi** [reply]
yes

**Reply by ghost** [reply]
this is a very good news 😃 .
Is there a release date?

**Reply by rdementi** [reply]
next week it should be in the master branch

**Reply by ghost** [reply]
thanks  😉

**Reply by rdementi** [reply]
it is in the master branch now

**Reply by XeLoRy** [reply]
So reactive, thanks a lot. It will be interesting to show you one day the business use cases we developing using PCM (a startup project). It is a shame there is not a startup program at Intel ;) !
